utils/route-utils.ts

```diff
@@ -5,23 +5,25 @@ const defaultQty = 10;
 const imageWidthDefault = 640;
 const imageHeightDefault = 480;
 
-const getQtyFromRequest = (request : Request | null, overrideDefaultQty?: number | null) : number => {
-    return request.params.qty ? parseInt(request.params.qty) : (overrideDefaultQty? overrideDefaultQty: defaultQty);
-}
+const getQtyFromRequest = (request: Request | null, overrideDefaultQty?: number | null): number => {
+  var value = parseInt(request.params.qty);
```

Comment:
Hi @siddharth1729 , thank you for the work on this PR. for the max qty fix we are going to use the code that is in the following PR. https://github.com/ageddesi/Mocked-API/pull/142/files